.travis.yml

```diff
@@ -147,8 +145,28 @@ install:
 script:
   - echo $PWD
   - if [[ -z "$QEMU_PLATFORM" ]]; then make check ; fi
-  - if [[ "$QEMU_PLATFORM" ]]; then docker build  --build-arg QEMU_PLATFORM --build-arg QEMU_OS --file docker/images/test-arm/Dockerfile  . -t skydev-test ; fi
-
+  - if [[ "$QEMU_PLATFORM" ]]; then
+    docker build  --build-arg QEMU_PLATFORM --build-arg QEMU_OS --file docker/images/test-arm/Dockerfile  . -t skydev-test ;
+    fi
+before_deploy:
+  - export VERSION="$(git describe --tags --exact-match HEAD 2> /dev/null)"
+  - export ARCH="$(uname -m)"
```

Comment:
@stdevHan this is not accurate . If `QEMU*` vars set then before deploy steps have to run inside `balenalib` Docker containers .